<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindAndReplaceDialog</class>
 <widget class="QDialog" name="FindAndReplaceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Find and Replace</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Find:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="3">
      <widget class="QLineEdit" name="text_find"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Replace:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="3">
      <widget class="QLineEdit" name="text_replace"/>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QCheckBox" name="case_sensitive">
       <property name="text">
        <string>Case sensitive</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2" colspan="2">
      <widget class="QCheckBox" name="whole_words">
       <property name="text">
        <string>Whole words only</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="button_find_next">
       <property name="text">
        <string>Find Next</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_find_previous">
       <property name="text">
        <string>Find Previous</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_replace">
       <property name="text">
        <string>Replace</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_replace_all">
       <property name="text">
        <string>Replace All</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="button_close">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_close</sender>
   <signal>clicked()</signal>
   <receiver>FindAndReplaceDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
